control/views.py

```diff
@@ -258,3 +258,36 @@ def weight_converter(request):
 
     template = loader.get_template('control/generic_converter.html')
     return HttpResponse(template.render(request=request, context=context))
+
+
+SPEED_VALUES = {
+    'miles per hour': 1,
+    'foot per second': 1.46667,
+    'metre per second': 0.44704,
+    'kilometre per hour': 1.60934,
```

Comment:
Change the keys, replacing the spaces by `_`: `kilometre_per_hour`

Reply:
will do ;)